main/contracts/index.ts

```diff
@@ -1,24 +1,60 @@
 import log from 'electron-log'
-import fetch from 'node-fetch'
-import { Interface } from '@ethersproject/abi'
-import erc20 from '../externalData/balances/erc-20-abi'
-
-const erc20Abi = JSON.stringify(erc20)
+import fetch, { Response } from 'node-fetch'
+import { Interface, JsonFragment } from '@ethersproject/abi'
+import { hexToNumberString } from 'web3-utils'
 
 interface EtherscanSourceCodeResponse {
   status: string,
   message: string,
   result: ContractSourceCodeResult[]
 }
 
+interface SourcifySourceCodeResponse {
```

Comment:
as we add more sources for contract data, lets put them in their own modules as they have their own API implementations